assets/src/blocks/Gallery/GalleryFrontend.js

```diff
@@ -16,35 +16,31 @@ const imagesToItems = images => images.map(
 );
 
 export const GalleryFrontend = ({
-  gallery_block_title,
-  gallery_block_description,
-  className,
-  gallery_block_style,
-  images,
+  attributes,
+  renderLightbox = false,
 }) => {
-  const layout = getGalleryLayout(className, gallery_block_style);
+  const className = attributes.className ?? '';
+  const layout = getGalleryLayout(className, attributes.gallery_block_style ?? '');
   const postType = document.body.getAttribute('data-post-type');
-
-  const { isOpen, index, openLightbox, closeLightbox } = useLightbox();
-
+  const images = attributes.images ? attributes.images : [];
   const items = imagesToItems(images);
+  const { isOpen, index, openLightbox, closeLightbox } = useLightbox();
 
   return (
-    <section className={`block ${GALLERY_BLOCK_CLASSES[layout]} ${className ?? ''}`}>
-      {gallery_block_title &&
+    <section className={`block ${GALLERY_BLOCK_CLASSES[layout]} ${className}`}>
+      {attributes.gallery_block_title &&
         <header>
-          <h2 className="page-section-header" dangerouslySetInnerHTML={{ __html: gallery_block_title }} />
+          <h2 className="page-section-header" dangerouslySetInnerHTML={{ __html: attributes.gallery_block_title }} />
         </header>
       }
-
-      {gallery_block_description &&
-        <div className="page-section-description" dangerouslySetInnerHTML={{ __html: gallery_block_description }} />
+      {attributes.gallery_block_description &&
+        <div className="page-section-description" dangerouslySetInnerHTML={{ __html: attributes.gallery_block_description }} />
       }
-      {layout === 'slider' && <GalleryCarousel onImageClick={openLightbox} images={images || []} />}
-      {layout === 'three-columns' && <GalleryThreeColumns onImageClick={openLightbox} images={images || []} postType={postType} />}
-      {layout === 'grid' && <GalleryGrid onImageClick={openLightbox} images={images || []} />}
+      {layout === 'slider' && <GalleryCarousel onImageClick={openLightbox} images={images} />}
+      {layout === 'three-columns' && <GalleryThreeColumns onImageClick={openLightbox} images={images} postType={postType} />}
+      {layout === 'grid' && <GalleryGrid onImageClick={openLightbox} images={images} />}
 
-      <Lightbox isOpen={isOpen} index={index} items={items} onClose={closeLightbox} />
+      {renderLightbox && <Lightbox isOpen={isOpen} index={index} items={items} onClose={closeLightbox} />}
```

Comment:
Checking this, we'll avoid to have SSR conflicts between React Elements. For example the `React Portal`, which is used by the [Lightbox library](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/932/files#diff-7603fc2cbf7cc1eec26221206dffa1f05c92e5b18163c0ab259a2b83443d1767R59).

And this implementation is mostly related to [this other functionality](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/932/files#diff-ea83b4bc3c4de5b445d5e8cc611f40046841a40eb55e6205430201302584c4dcR16).